content/id/docs/reference/glossary/selector.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Selector
+id: selector
+date: 2018-04-12
+full_link: /id/docs/concepts/overview/working-with-objects/labels/
+short_description: >
+  Memungkinkan pengguna untuk memfilter daftar sumber daya berdasarkan label yang digunakan.
+
+aka:
+tags:
+- fundamental
+---
+Memungkinkan pengguna untuk memfilter daftar sumber daya berdasarkan label yang digunakan.
+
+<!--more--> 
+
+_Selector_ diterapkan ketika melakukan kueri terhadap daftar sumber daya untuk memfilter mereka berdasarkan labelnya.
```

Comment:
"memfilter" = "menyaring"?
```suggestion
_Selector_ diterapkan ketika melakukan kueri terhadap daftar sumber daya untuk menyaring mereka berdasarkan labelnya.
```

Reply:
memfilter is a valid kbbi https://kbbi.web.id/filter, and has been used in other localisations too, e.g. https://support.microsoft.com/id-id/office/memfilter-data-dalam-rentang-atau-tabel-01832226-31b5-4568-8806-38c37dcc180e